docs/source/documentation/data-products/components/product-specification.html.md.erb

```diff
@@ -140,6 +158,11 @@ Most Data Products will be tagged as `production` - the other statuses will flag
 <!--
 - `allow-derived-products` - typically `true` (and `true` if not supplied), but in some cases you may not wish derived products to be created. If the product status attribute is "production", you may be asked to justify disallowing derived products as we want to encourage data reuse. -->
 
+<!-- (Alpha) -->
+### Contact information
+
+You must supply an `email` address to indicate the point of contact between consumers and maintainers of the Data Product. It could be the owner or a distribution list, but must be reliable and responsive.
```

Comment:
This is in contrast to the owner email, which we're assuming is the email that matches the one the data owner signs in with (@justice.gov.uk).

Should make sure this is clear as people might wonder why they have to provide two emails.